app-modules/student-data-model/src/Filament/Widgets/StudentStats.php

```diff
@@ -66,7 +66,7 @@ protected function getStats(): array
                 })),
             Stat::make('My Alerts', Cache::tags(['students', "user-{$user->getKey()}-student-alerts"])
                 ->remember("user-{$user->getKey()}-student-alerts-count", now()->addHour(), function () use ($user): int {
-                    return $user->studentAlerts()->status(AlertStatus::Active)->count();
+                    return $user->studentAlerts()->status(SystemAlertStatusClassification::Active)->count();
```

Comment:
This should be updated to check the relationship or this old way based on the Feature Flag